testing/op-l2.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SEQ_KEY="73bead3e3e588be3ae77598160d6a5017acdb008973f21166571e400a4b51246"
+L1_RPC="http://localhost:8545"
+RPC_KIND="any"
```

Comment:
`RPC_KIND` should be `basic` to avoid calling non existing methods into the Polaris JSON RPC API

Reply:
fixed